### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by ashimali**
Basic error pages for 401, 404, 500 errors

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -116,3 +117,13 @@ def useful_headers_after_request(response):
...
+
+
+def register_errorhandlers(application):
+    def render_error(error):
+        # If a HTTPException, pull the `code` attribute; default to 500
+        error_code = getattr(error, 'code', 500)
+        return render_template("error/{0}.html".format(error_code)), error_code
+    for errcode in [401, 404, 500]:
+        application.errorhandler(errcode)(render_error)
+    return None
```
Comment (mannickutd) [inline app/__init__.py]: None is redundant

Comment (ashimali) [inline app/__init__.py]: Good point

File: app/templates/error/404.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% extends "admin_template.html" %}
+{% block page_title %}Page Not Found{% endblock %}
```
Comment (quis) [inline app/templates/error/404.html]: > Page **n**ot **f**ound

**Comment by quis** [comment]
Other than one remarkably nitpicky comment, this looks good to me.

**Comment by ashimali** [comment]
@quis no not nitpicky, I think you are right on that one :)

**Pull request merged**